<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>416</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>537</width>
    <height>359</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidgetUsers">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>18</y>
      <width>170</width>
      <height>312</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelUsers">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>1</y>
      <width>78</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Пользователи</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelChat">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>1</y>
      <width>78</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Чат</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditMessages">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>18</y>
      <width>350</width>
      <height>312</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSend">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>335</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключиться</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>336</y>
      <width>350</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите имя</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxReceiver">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>389</y>
      <width>170</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditMessage">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>361</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите сообщение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCall">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>361</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Позвонить</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
